web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1167,8 +1174,8 @@ fixmystreet.update_report_a_problem_btn = function() {
 
 fixmystreet.update_public_councils_text = function(text, bodies) {
     bodies = bodies.join('</strong> ' + translation_strings.or + ' <strong>');
-    text = text.replace(/<strong>.*<\/strong>/, '<strong>' + bodies + '</strong>');
-    $('#js-councils_text').html(text);
+    bodies = '<strong>' + bodies + '</strong>';
+    $('.js-councils_text_names').html(bodies);
```

Comment:
This breaks the existing functionality (bad that it doesn't break a test then!) in that the `text` passed in might change more than just the bodies - e.g. it might now be the unresponsive text where it then says it's published online but not sent anywhere. Only replacing the bodies loses this. But as per my other comments, I think the server can return the correct text and this function can remain the same.

Reply:
6faa885a0e530462c4d393f69b29982e27a116b7 should hopefully fix this!